clients/vscode-hlasmplugin/package.json

```diff
@@ -187,7 +188,7 @@
       "type": "object",
       "title": "hlasmplugin configuration",
       "properties": {
-        "semanticHighlightingColors": {
+        "hlasmplugin.semanticHighlightingColors": {
```

Comment:
Are we still using name hlasmplugin? Maybe just hlasm or HlasmLanguageSupport

Reply:
all other properties have that name. but simply 'hlasm' seems more appropriate